src/watcher.rs

```diff
@@ -38,8 +38,16 @@ impl WatcherOpts {
     /// This extracts the bits of an [`Opts`] struct relevant to the [`run_watcher`] session
     /// without cloning or taking ownership of the entire thing.
     pub fn from_cli(opts: &Opts) -> Self {
+        let watch = if let Some(file) = &opts.file {
+            let mut paths = opts.watch.paths.clone();
+            paths.push(file.clone());
+            paths
+        } else {
+            opts.watch.paths.clone()
+        };
```

Comment:
Let's put this in `Opts::init`:

https://github.com/MercuryTechnologies/ghciwatch/blob/2c4b72ca870559134a234831c6c59563a34631d9/src/cli.rs#L235-L237